src/main/java/seedu/address/model/card/Card.java

```diff
@@ -27,19 +28,28 @@
     private final UUID id;
     private final String front;
     private final String back;
+    private final Calendar nextReview;
+    private final int lastInterval;
```

Comment:
`lastInterval` and `easingFactor` should probably go into its own class. Is `lastInterval` a fixed number?

Reply:
last interval changes